### ISSUE TITLE
[FEATURE] Better Brand specific components

### ISSUE THREAD

**Issue by vogloblinsky** [Orange-OpenSource/Orange-Boosted-Bootstrap#653]
Comparing official Boostrap website and Boosted, it is not very clear which components are specific to Orange brand and default boostrap. For example stepbar.
The naming are not very well defined : Navbar Orange, Orange megamenu, Orange footer
May be another category in left menu could be fine ?

**Comment by Lausselloic** [comment]
Maybe not another category, but prefix all custom component by Orange? to Group them in left navigation. A complete list of [custom component is availabe](https://boosted.orange.com/docs/4.6/about/boosted-history/#custom-components)

**Comment by julien-deramond** [comment]
Contrary to Boosted v4 where the naming was inconsistent, we chose in Boosted v5 to avoid using "Orange" for the components' naming; except for "Orange navbar" to avoid the collision with the "Navbar" component from Bootstrap.

In our opinion, it's easier to search, to find in the side menu to avoid having 5 "Orange something" components not easily foundable when ordered alphabetically. On top of that, our objective is to have a multi-brand approach at some point, so the naming/the doc should probably tend to be brand-agnostic.

It's a bit hidden in the documentation, but there's a complete [list in the About section of Boosted-specific components](https://boosted.orange.com/docs/5.2/about/overview/#custom-components) (IMO "custom" is not the right word) as mentioned by Loïc.

Closing it for now but we can revisit later if needed.
